lizard_efcis/urls.py

```diff
@@ -26,6 +26,10 @@
         views.LinesAPI.as_view(),
         name='efcis-lines'),
 
+    url(r'^parametergroeps/$',
+        views.ParameterGroepAPI.as_view(),
+        name='parametergroep-tree'),
```

Comment:
Ik prefix altijd alles met de pakketnaam (dus `efcis-parametergroep-tree` in dit geval).

Anders krijg je meerdere 'home' pagina's enzo.